src/resources/filters/quarto-post/typst.lua

```diff
@@ -120,9 +120,20 @@ function render_typst_fixups()
       return div
     end,
     Table = function(tbl)
-      -- https://github.com/quarto-dev/quarto-cli/issues/10438
+      -- Check if table has already been processed
+      if tbl.classes:includes("typst-processed") then
+        return tbl
```

Comment:
I believe this is still going to fail. You need to stop the topdown traversal with a `return tbl, false` statement. Refer to the Pandoc docs for more: https://pandoc.org/lua-filters.html#topdown-traversal